CHANGELOG.md

```diff
@@ -2,14 +2,47 @@
 
 Notable changes to QMCPACK will be documented in this file.
 
+## [Unreleased]
+
+* Size consistent t-moves implemented (Casula 2010 algorithm).
+  Enabled via nonlocalmoves parameter, see manual.
+* Bugfix: For non-git builds, build process failed on some systems
+  due to git-rev.h handling.
+* Optimized load balancing in DMC. Command line option async_swap
+  removed. Parameter use_nonblocking now disables non-blocking MPI
+  load balancing.
+* Improved memory handling and usage in SoA code, increases
+  performance.  
+* Improved stability of GPU matrix inversion for large runs.
+* Ongoing improvements to output to improve readability.
+* Initial interface to PySCF for real space QMC trial wavefunctions.
+* Enabled use of HDF5 files for Gaussian based wavefunctions
+  with SoA implementation.
+* Added Appendix to manual listing all known QMCPACK publications.
+  This will be updated on an ongoing basis. Please advise of any
+  missing publications.
+* Improved Momentum distribution estimator: optimized and
```

Comment:
and ...?

Reply:
Fixed. Was thinking about the bugfix, but that was in 3.3.0.